bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -216,6 +221,7 @@ impl From<matrix_sdk::TransmissionProgress> for TransmissionProgress {
 pub struct Client {
     pub(crate) inner: AsyncRuntimeDropped<MatrixClient>,
     delegate: RwLock<Option<Arc<dyn ClientDelegate>>>,
+    utd_hook_manager: Option<Arc<UtdHookManager>>,
```

Comment:
Ah sorry, there's been a misunderstanding here: I was suggesting to move the `UtdHookManager` into the *SDK* `Client` (likely in the `ClientInner`), not the *FFI* `Client`. This way, you could avoid having to pass the `UtdHookManager` to every room you're creating in the next commit.

Reply:
Well I can do that as well but I would handle it as a separate PR, in the spirit of keeping it rather contained. The goal here was to simplify how we deal with timelines for which this should do the trick, no?